src/PIL/TiffImagePlugin.py

```diff
@@ -169,6 +169,7 @@
     (II, 0, (1,), 2, (8,), ()): ("L", "L;IR"),
     (MM, 0, (1,), 2, (8,), ()): ("L", "L;IR"),
     (II, 1, (1,), 1, (8,), ()): ("L", "L"),
+    (II, 0, (1,), 1, (16,), ()): ("I", "I;16"),
```

Comment:
```suggestion
    (II, 0, (1,), 1, (16,), ()): ("I;16", "I;16"),
```
You will notice that a few lines later, there is a rawmode of "I;16".
https://github.com/python-pillow/Pillow/blob/243402e78e2bf2b7af478c6d891816e372b5c3f9/src/PIL/TiffImagePlugin.py#L176
and the mode used is "I;16". It would seem more consistent to also use it here.

Reply:
Moving the line to another position would also be helpful. I've created https://github.com/JayWiz/Pillow/pull/1 for this.